unittest/bindings/testutils.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
```

Comment:
Remove this file. These functions are in `test_utils.py`, excluding `df_dx` and `df_dq` because they are not needed.